custom/src/main/java/com/splunk/opentelemetry/jaeger/JaegerThriftSpanExporterFactory.java

```diff
@@ -33,17 +33,11 @@
 
   static final String SIGNALFX_AUTH_TOKEN = "signalfx.auth.token";
   public static final String OTEL_EXPORTER_JAEGER_ENDPOINT = "otel.exporter.jaeger.endpoint";
-  static final String OTEL_EXPORTER_JAEGER_SERVICE_NAME = "otel.exporter.jaeger.service.name";
 
   @Override
   public SpanExporter createExporter(ConfigProperties config) {
     JaegerThriftSpanExporterBuilder builder = JaegerThriftSpanExporter.builder();
 
-    String serviceName = config.getString(OTEL_EXPORTER_JAEGER_SERVICE_NAME);
```

Comment:
if we wanted to support this for a release, we could probably add this value to the resource that we configure into the SDK. I don't know how important that is, but we can probably make it work if we have to.

Reply:
Yep, an additional `ResourceProvider` would work. The question is, do we really need that? We've already had lots of breaking changes in configuration in this release.